tests/features/scheduler.feature

```diff
@@ -10,28 +10,32 @@ Feature: Scheduler features
   Scenario: User can schedule a date to publish a content
     When I go to "node/add/page"
```

Comment:
The tests are fine but we are missing the tests where we publish a new revision of an existing node. We also need a tests that shows how a node with an incorrect state is not published when the date comes.

Reply:
Added two new scenarios